packages/ns-storage/files/30_ns-storage

```diff
@@ -18,6 +37,11 @@ config syslog 'syslog'
 config selector ns_memory
 	option source '*.*'
 	option destination ':omfile:\$log_rotation'
+
+config log_rotator 'ns_log'
+	option log_file '/var/log/messages'
+	option log_size '$log_size'
+	option rotate_command '/usr/sbin/rotate-messages'
```

Comment:
While this seems a nice idea, we are trying to keep the package in sync with upstream which does not have such support inside the init.d.
I eventually move this to an upstream patch.